From d64aedb6bb1f74eb0334d552dc5586a18418ed9f Mon Sep 17 00:00:00 2001
From: Andy Fleming <afleming@freescale.com>
Date: Mon, 8 Apr 2013 18:35:03 -0500
Subject: [PATCH 065/987] dpaa: Fix dependencies for FMAN and DPAA Ethernet

[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]

BMAN and QMAN are merged, and both the DPAA Ethernet and FMAN
drivers require the QMAN/BMAN support to build. We therefore
note that dependency in the Kconfigs.

Change-Id: I7eda3375ca0e9e760cd7650be08c9b41acaac9af
Signed-off-by: Andy Fleming <afleming@freescale.com>
Reviewed-on: http://git.am.freescale.net:8181/1112
Reviewed-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
Tested-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
---
 drivers/net/ethernet/freescale/dpa/Kconfig  | 3 +--
 drivers/net/ethernet/freescale/fman/Kconfig | 2 +-
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/drivers/net/ethernet/freescale/dpa/Kconfig b/drivers/net/ethernet/freescale/dpa/Kconfig
index 60e653f..148adc5 100644
--- a/drivers/net/ethernet/freescale/dpa/Kconfig
+++ b/drivers/net/ethernet/freescale/dpa/Kconfig
@@ -1,7 +1,6 @@
 config DPA_ETH
 	bool "DPAA Ethernet"
-# FIXME enable this when all dependencies are merged
-#	depends on FSL_SOC && FSL_BMAN && FSL_QMAN && FSL_FMAN
+	depends on FSL_SOC && FSL_BMAN && FSL_QMAN && FSL_FMAN
 	select PHYLIB
 	---help---
 	  Data Path Acceleration Architecture Ethernet driver, supporting the
diff --git a/drivers/net/ethernet/freescale/fman/Kconfig b/drivers/net/ethernet/freescale/fman/Kconfig
index e640465..d3595010 100644
--- a/drivers/net/ethernet/freescale/fman/Kconfig
+++ b/drivers/net/ethernet/freescale/fman/Kconfig
@@ -2,7 +2,7 @@ menu "Frame Manager support"
 
 menuconfig FSL_FMAN
 	bool "Freescale Frame Manager (datapath) support"
-	# depends on PPC_E500MC
+	depends on FSL_SOC && FSL_BMAN && FSL_QMAN
 	default y
 	---help---
 		If unsure, say Y.
-- 
1.9.1

